transport-native-epoll/src/main/java/io/netty/channel/epoll/EpollSocketChannel.java

```diff
@@ -39,6 +43,7 @@
 
     private volatile InetSocketAddress local;
     private volatile InetSocketAddress remote;
+    private volatile Set<InetAddress> tcpMd5Sig = Collections.emptySet();
```

Comment:
Could this field have a better name? It does not contains any signature but a set of addresses only.